### DISCUSSION TITLE
Grouping bits into bus ?

### DISCUSSION THREAD

**Discussion by ba38** [hneemann/Digital#1368] [General]
Hi All
And thanks for this very nice software.
I have design a circuit which is built from single bits  (outputs S0,..S7 for instance) and I would like to group them into a bus S[7..0]. It would be simpler for designing a test file (or several ones) and much less prone to errors.
I don't find either in the menus, neither in the help file.
How can I do it ?
Thanks a lot.
Bernard

**Comment by Slicudis** [comment]
Use a splitter/merger
![image](https://github.com/user-attachments/assets/4a79d849-642e-4ccb-8fce-dc90444ed6e9)
